app/views/news/2016-09-30-september-2016-monthly-update.html.md

```diff
@@ -0,0 +1,59 @@
+---
+summary: "Ruby Together paid for 71.3 hours of work, and had 18 new members join. This month we talked about a Year of Ruby Together, released Bundler 1.13, 1.13.1, 1.13.2, and generally got very engaged with the Ruby community."
+---
+
+<% title "September 2016 Monthly Update" %>
+
+Welcome to the September 2016 Ruby Together monthly update! This month we: added a variety of new features in Bundler 1.13, and bug fixes in 1.13.1 and 1.13.2. Paid for 71.3 hours of work on Bundler, RubyGems, and RubyGems.org. And gave a great talk at EuRuKo 2016 leading to 18 (!) new members.
+
+## ruby together news
+
+André did a [1 year recap](https://rubytogether.org/news/2016-09-27-a-year-of-ruby-together) of working on Ruby, Together. That post was also given in [talk form](https://speakerdeck.com/indirect/a-year-of-ruby-together) at EuRuKo 2016. In it, he discusses scaling, volunteer work, hacks, and open source.
+
+(( André write this part!!! ))
+
+## bundler news
+
+Early in the month, we released [Bundler 1.13](http://bundler.io/blog/2016/09/08/bundler-1-13.html). This update features `required_ruby_version`, `bundle doctor`, and platform `lock` commands. Later in the month, [Bundler 1.13.1](https://github.com/bundler/bundler/milestone/32?closed=1) shipped with a variety of minor bug fixes.
+
+Chris Morris ([@chrismo](https://github.com/chrismo)) was added to Bundler's [contributor team](http://bundler.io/contributors.html). His core team promotion was sponsored by the experimental [conservative update](http://bundler.io/blog/#experimental-conservative-updates) that shipped with 1.13.
+
+We fixed a [GEM_PATH regression](https://github.com/bundler/bundler/pull/4992) in release 1.13. This bug was truly an adventure, and threw our core team for a loop for most of two weeks. Finally, we started working towards the Better Platform Support update with [the help](https://github.com/bundler/bundler/issues/4984) of [@headius](https://github.com/headius) and [jruby](http://jruby.org/).
+
+In total, Bundler had 37 merged pull requests and 83 commits from 11 authors, and closed 50 issues.
+
+## rubygems.org news
+
+We experienced the [life and death](https://github.com/rubygems/rubygems.org/issues/1429) of [@rubygems](https://twitter.com/rubygems) this month. Many thanks to our valiant users, always on hand to pull our Twitter accounts back from the abyss.
+
+We also worked on both server side ([rate limiting](https://github.com/rubygems/rubygems.org/pull/1414)), and end user focused ([gem yank](https://github.com/rubygems/rubygems.org/pull/1396)) security issues.
+
+In total, RubyGems.org had 13 merged pull requests and 26 commits from 5 authors.
+
+## rubygems news
+
+We released [the last few months of changes](https://github.com/rubygems/rubygems/pulls?utf8=%E2%9C%93&q=merged%3A2016-06-22..2016-09-26) as RubyGems 2.6.7. This release features bug fixes for `--user-install`, and some initial bundler integration.
```

Comment:
this release actually only had a few bug fixes

https://github.com/rubygems/rubygems/compare/v2.6.6...v2.6.7